pool/web/view/page/page_admin_dashboard.ml

```diff
@@ -75,8 +75,11 @@ let index
         ; Partials.upcoming_sessions_list language upcoming_sessions
         ]
     in
-    let elements = [ session_list; calendar_html ] in
-    let html = if recruiter_layout then CCList.rev elements else elements in
+    let html =
+      if recruiter_layout
+      then [ calendar_html ]
+      else [ session_list; calendar_html ]
```

Comment:
I recomment to solve it with a polymorphic variant, so we do not execute the sessions query for no reason. 

```
let html = match layout with
 | `Recruiter -> [ calendar_html ]
 | `Assistant sessions ->  build sessions .....
```